pkg/skaffold/initializer/builders.go

```diff
@@ -218,23 +217,7 @@ func resolveBuilderImages(builderConfigs []InitBuilder, images []string, force b
 		images = util.RemoveFromSlice(images, image)
 	}
 	if len(choices) > 0 {
-		logrus.Warnf("unused builder configs found in repository: %v", choices)
+		logrus.Warnf("unused builder kubernetesManifests found in repository: %v", choices)
```

Comment:
Not sure I understand this message, as a user.

Reply:
accidental comment refactoring, reverted!